README.md

```diff
@@ -1,15 +1,65 @@
-# ROS2 Wrapper for Intel&reg; RealSense&trade; Devices
-These are packages for using Intel RealSense cameras (D400, L500 and SR300 cameras) with ROS2.
-
-This version supports ROS2 Dashing, Eloquent, Foxy, Galactic and Rolling.
-
-LibRealSense supported version: v2.51.1 (see [realsense2_camera release notes](https://github.com/IntelRealSense/realsense-ros/releases))
-
-
-## For LibRS ROS1 Wrapper please refer to [ROS1-legacy branch](https://github.com/IntelRealSense/realsense-ros/tree/ros1-legacy)
-
-## Please notice: if you are moving from RealSense [ROS2-legacy branch](https://github.com/IntelRealSense/realsense-ros/tree/ros2-legacy) to ROS2-development:
-- **Changed Parameters**:
+<h1 align="center">
+   <img src="https://www.intelrealsense.com/wp-content/uploads/2020/09/intel-realsense-logo-360px.png" alt="Intel® RealSense™" title="Intel® RealSense™" />
+</h1>
+
+<p align="center">
+  ROS2 packages for using Intel RealSense D400 cameras.<br>
+  <a href="https://github.com/IntelRealSense/realsense-ros/releases">Latest release notes</a>
+</p>
+
+<hr>
+
+<div style="text-align:center">
+
+[![rolling][rolling-badge]][rolling]
+[![humble][humble-badge]][humble]
+[![foxy][foxy-badge]][foxy]
+[![galactic][galactic-badge]][galactic]
+[![eloquent][eloquent-badge]][eloquent]

```

Comment:
Lets remove dashing and eloquent,  also ubuntu 18

Reply:
done